swarm/network/stream/delivery.go

```diff
@@ -255,7 +255,7 @@ func (d *Delivery) RequestFromPeers(ctx context.Context, req *network.Request) (
 			}
 			sp = d.getPeer(id)
 			if sp == nil {
-				log.Warn("Delivery.RequestFromPeers: peer not found", "id", id)
+				//log.Warn("Delivery.RequestFromPeers: peer not found", "id", id)
```

Comment:
lets leave this. this warning occurring too many times is indicating a bug.

Reply:
I understand that this is a bug, but this is a very spammy log line, and it is not helpful.

![screen shot 2018-11-21 at 12 48 53](https://user-images.githubusercontent.com/50459/48839468-e7ccdd00-ed8b-11e8-9c63-5271a7070ea4.png)

400k times for a single peer on a single host. Users are complaining that this is filling their disks with logs, and rightly so.

---

I am currently looking into it, but we should not have another release with this spammy log.

The peer in question in the logs is actually present in the peer list, so I am not sure why `d.getPeer(id)` is returning nil.

Reply:
If it's a bug and we just silently "ignore" it, then we should open an issue to investigate it and find a solution.